### PR TITLE
Fixes some links

### LINKED ISSUE #388: Projects hero links missing
http://science-mozilla-org-staging.herokuapp.com/projects

The links in `Have a project you want to add?Bring it to our Global Sprint, Mozfest, or Working Open Workshop.` need wiring up.

Also there should be a space after the `?`.

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #388

### CODE REVIEW

File: app/react/pages/projects/projects.jsx [hunk 1]
```diff
@@ -91,7 +92,7 @@ export default React.createClass({
         <div className="jumbotron text-xs-center jumbotron-fluid m-b-0">
           <h2>Projects</h2>
           <p className="lead m-t-1">A curated collection of open source, web-based tools to help you do better science.</p>
-          <p>Have a project you want to add?Bring it to our <a href="#">Global Sprint</a>, <a href="#">Mozfest</a>, or <a href="#">Working Open Workshop</a>.</p>
+          <p>Have a project you want to add? Bring it to our <Link to="programs/events/global-sprint-2016">Global Sprint</Link>, <a href="https://mozillafestival.org">Mozfest</a>, or <a href="#">Working Open Workshop</a>.</p>
```
Comment (gvn) [inline app/react/pages/projects/projects.jsx]: Can you add `target="_blank"` to the off-site links?

**Comment by gvn** [comment]
1 nit and R+

File: app/react/pages/projects/projects.jsx (hunk 1 above)
Comment (alanmoo) [inline app/react/pages/projects/projects.jsx]: I thought it felt kind of odd being inconsistent across the 3 links in this area...

**Comment by alanmoo** [comment]
@gvn one more look? Added the link and target.

**Comment by gvn** [comment]
Perfect 👍

**Pull request merged**